.github/workflows/cd.yml

```diff
@@ -0,0 +1,185 @@
+name: Sign and prerelease
+
+# Run the workflow on when code or a semver tag is pushed to main branch,
+# and on pull requests towards main branch
+on:
+  push:
+    branches:
+      - 'main'
+    tags:
+      # semver, e.g. 1.2.0 (does no match 0.1.2)
+      - '[1-9]+.[0-9]+.[0-9]+'
+      # semver with prerelease info, e.g. 1.0.2-beta.1 or 1.2.3-rc.10
+      - '[1-9]+.[0-9]+.[0-9]+-[a-z]+.[0-9]+'
+      # do not match prerelease starting w/ 0, e.g. 1.0.2-beta.0 or 1.2.3-rc.01
+      - '![1-9]+.[0-9]+.[0-9]+-[a-z]+.[0]*'
+      # semver with date info, e.g. 1.0.2-20221125
+      - '[1-9]+.[0-9]+.[0-9]+-[0-9]+'
+      # do not match date starting w/ 0, e.g. 1.0.2-01232023
+      - '![1-9]+.[0-9]+.[0-9]+-[0]*'
+  pull_request:
+    branches:
+      - 'main'
+
+jobs:
+  build:
+    if: (github.event_name != 'pull_request')
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: ["armv7hf", "aarch64"]
+    outputs:
+      EAP_FILE: ${{ steps.get_eap_file_name.outputs.EAP_FILE }}
+      EAP_FILE_ARMV7HF: ${{ steps.save_full_file_name.outputs.EAP_FILE_ARMV7HF }}
+      EAP_FILE_AARCH64: ${{ steps.save_full_file_name.outputs.EAP_FILE_AARCH64 }}
+    env:
+      exrepo: build-test
+      exname: docker-acap
+      DOCKER_HUB_REPOSITORY: 'axisecp/docker-acap'
+    steps:
+        # sha for actions/checkout@v3 at time of commit.
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+      - name: Create base image metadata
+        id: meta
+        uses: ./.github/actions/metadata-action
+        with:
+          suffix: -${{ matrix.arch }}
+          repository: ${{ env.DOCKER_HUB_REPOSITORY }}
+          get_version: 'true'
+      - name: Update manifest file
+        if: ( github.ref_type == 'tag')
+        uses: ./.github/actions/update-acap-manifest-action
+        with:
+          manifest_file: ./manifest-${{ matrix.arch }}.json
```

Comment:
Update here: there is only one manifest.json in this repo and it is in the app-folder